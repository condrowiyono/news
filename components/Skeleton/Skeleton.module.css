.skeleton {
  background: #BDBDBD ;
  margin-bottom: 4px;
}