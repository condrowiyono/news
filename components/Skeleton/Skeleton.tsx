import style from "./Skeleton.module.css";

interface ISkeleton {
  width: number;
  height: number;
  borderRadius?: number;
}

const Skeleton = ({ width, height, borderRadius = 4 }: ISkeleton) => {
  return (
    <div
      className={style.skeleton}
      style={{ height, width, borderRadius }}
      data-testid="skeleton"
    />
  );
};

export default Skeleton;
