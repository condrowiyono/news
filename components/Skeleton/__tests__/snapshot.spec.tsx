import { render } from "@testing-library/react";
import Skeleton from "@components/Skeleton";

it("renders Button unchanged", () => {
  const { container } = render(<Skeleton width={100} height={300} />);
  expect(container).toMatchSnapshot();
});
