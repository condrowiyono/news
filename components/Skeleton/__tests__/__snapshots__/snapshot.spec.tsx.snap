// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button unchanged 1`] = `
<div>
  <div
    class="skeleton"
    data-testid="skeleton"
    style="height: 300px; width: 100px; border-radius: 4px;"
  />
</div>
`;
